frontend/src/scenes/events/eventsTableLogic.ts

```diff
@@ -245,29 +237,35 @@ export const eventsTableLogic = kea<eventsTableLogicType<ApiError, EventsTableLo
         },
     }),
 
-    urlToAction: ({ actions, values }) => ({
-        '*': (_, searchParams) => {
-            try {
-                // if the url changed, but we are not anymore on the page we were at when the logic was mounted
-                if (router.values.location.pathname !== values.initialPathname) {
+    urlToAction: ({ actions, values, props }) => {
+        return {
+            [props.sceneUrl]: (_: Record<string, any>, searchParams: Record<string, any>): void => {
+                try {
+                    // if the url changed, but we are not anymore on the page we were at when the logic was mounted
+                    if (router.values.location.pathname !== values.initialPathname) {
```

Comment:
This `initialPathname` wouldn't work anymore with persons if the logic would remain mounted between scenes, as changing a person's ID will change the URL, and if this logic remains mounted, the `initialPathname` will point to somebody else... and the table will stop working.

Luckily we do unmount this logic for most persons, but there might be a day when we don't. I'd just remove this entire check. Now that you use `props.sceneUrl`, it doesn't make much sense anymore.